azuread/table_azuread_identity_provider.go

```diff
@@ -0,0 +1,72 @@
+package azuread
+
+import (
+	"context"
+
+	"github.com/manicminer/hamilton/msgraph"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureAdIdentityProvider() *plugin.Table {
+	return &plugin.Table{
+		Name:        "azuread_identity_provider",
+		Description: "Represents an Azure Active Directory (Azure AD) identity provider",
+		List: &plugin.ListConfig{
+			Hydrate: listAdIdentityProviders,
+			KeyColumns: plugin.KeyColumnSlice{
+				// Key fields
+				{Name: "id", Require: plugin.Optional},
+			},
+		},
+
+		Columns: []*plugin.Column{
+			{Name: "id", Type: proto.ColumnType_STRING, Description: "The unique identifier for the service principal.", Transform: transform.FromGo()},
+			{Name: "name", Type: proto.ColumnType_STRING, Description: "The display name for the service principal"},
```

Comment:
```suggestion
			{Name: "name", Type: proto.ColumnType_STRING, Description: "The display name for the service principal."},
```

All column descriptions should end in periods, please check over all other descriptions